Packs/CommonScripts/Scripts/HttpV2/HttpV2.yml

```diff
@@ -21,7 +20,10 @@ args:
   required: true
   secret: false
 - default: false
-  description: Specify a hash of headers to send with the request.
+  description: |-
+    Specify a hash of headers to send with the request.
+    Headers can be of string type but need to be formatted in the following ways:
+    `{"key": "value"}` or `"key": "value"`
```

Comment:
Please add an example with 2 values (so it'll be obvious that you pass more than 1 value using `,`.